Code/Legacy/CrySystem/Log.cpp

```diff
@@ -1501,10 +1474,15 @@ const char* CLog::GetModuleFilter()
 void CLog::FlushAndClose()
 {
 #if defined(KEEP_LOG_FILE_OPEN)
-    if (m_logFileHandle.IsOpen())
-    {
-        CloseLogFile();
-    }
+    m_logFileHandle.Flush();
+    CloseLogFile();
```

Comment:
The flush call isn't needed here since the close would flush the contents.

Reply:
True. Changed.